docs/docs/grafana.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Grafana
+weight: 11
+---
+
+The grafana Custom Resource Definition (CRD) exist to manage one or multiple managed and external grafana instances.
+
+## Grafana config
+
+In version 4 of the operator we defined all the configuration values that could be done in the CRD.
+In version 5 we don't longer do that, instead we supply you with `grafana.config` where you can add any
+grafana configuration values that you want.
+
+The operator do not make any extra validation of your configuration so just like Grafana your Grafana instance might be broken due to configuration error.
+
+To find all possible configuration options look at the [official documentation](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/).
+
+## Where should the operator look for Grafana resources?
+
+The operator needs lots of access to be able to manage Grafana instances.
+Depending on your needs you might want to limit the Grafana operator to only watch specific namespaces.
+
+Due to this we have given the operator 3 deployment modes.
+
+- cluster wide
+- single namespace
+- multiple namespaces
+
+To enter these specific modes you need to define the `WATCH_NAMESPACE` environment variable in the Grafana operator.
+How you do this will differ between the different deployment methods.
+
+`WATCH_NAMESPACE: ""`
+Will make the operator work cluster wide.
+
+`WATCH_NAMESPACE: "grafana"`
+Will make the operator only watch the grafana namespace, using this mode, makes it possible to use a role instead of a cluster role to give access to the operator.
+Depending on which deployment solution you use how to do this will differ.
+
+`WATCH_NAMESPACE: "grafana, monitor, foo, bar"`
+Will make the operator watch the following namespaces grafana, monitor, foo, bar. This will force you to still use the cluster wide rbac settings to enable the Grafana operator to monitor all of those namespaces.
+
+## External Grafana instances
+
+With the operator you can manage external Grafana instances.
+So even if you don't want to manage Grafana on your own you can still use dashboards as code.
+
+The `grafana.spec.external` config allows you to set an URL and provide authentication to talk to your Grafana instance.
+
+Since the operator do not own the external Grafana instance you are **not** able to send in any configuration values to grafana it self. So for example `grafana.spec.config` will not do anything if you want to manage an external Grafana instances.
```

Comment:
```suggestion
Since the operator does not own the external Grafana instance you are **not** able to send in any configuration values to grafana itself. So for example `grafana.spec.config` will not do anything if you want to manage an external Grafana instances.
```